eslzArm/managementGroupTemplates/policyDefinitions/policies.json

```diff
@@ -933,75 +947,139 @@
+                                                            },
+                                                            "parameters": {
+                                                                "routeTableName": {
+                                                                    "value": "[[parameters('routeTableName')]"
+                                                                },
+                                                                "vnetRegion": {
+                                                                    "value": "[[parameters('vnetRegion')]"
+                                                                },
+                                                                "requiredRoutes": {
+                                                                    "value": "[[parameters('requiredRoutes')]"
+                                                                },
+                                                                "disableBgpPropagation": {
+                                                                    "value": "[[parameters('disableBgpPropagation')]"
                                                                 }
-                                                            ]
+                                                            }
                                                         }
-                                                    }
-                                                ],
-                                                "outputs": {}
+                                                    }   
+                                                ]
+                                            },
+                                            "parameters": {
+                                                "routeTableName": {
+                                                "value": "[[parameters('routeTableName')]"
+                                                },
+                                                "vnetRegion": {
+                                                "value": "[[parameters('vnetRegion')]"
+                                                },
+                                                "requiredRoutes": {
+                                                "value": "[[parameters('requiredRoutes')]"
+                                                },
+                                                "disableBgpPropagation": {
+                                                "value": "[[parameters('disableBgpPropagation')]"
+                                                }
                                             }
                                         }
                                     }
                                 }
                             }
                         }
                     },
-                    "name": "Deploy-Default-Udr"
+                    "name": "Deploy-Custom-Route-Table"
```

Comment:
Is it intentional to give it a new name? That will result in a net-new policyDefinition from an ARM perspective. @krowlandson - thoughts on this from TF side?

Reply:
To @krowlandson's point, the intent was to implement a new policy that allows defining all the aspects of a spoke VNet's routing policy (UDRs and BGP propagation settings). Hence the new name and the "1.0.0" version. If I am missing anything that makes it difficult/not desirable to replace the existing policy, I am happy to prepare a PR that updates it, keeps the name and changes version to "1.1.0"

Reply:
Sounds good to me, replacing the one we had with a new, and keeping it as 1.0.0.